oauth2-test/src/main/java/org/springframework/security/test/oauth2/request/AccessTokenRequestPostProcessor.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.test.oauth2.request;
+
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.security.test.oauth2.support.AccessTokenAuthenticationBuilder;
+import org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.SecurityContextRequestPostProcessorSupport;
+import org.springframework.test.web.servlet.request.RequestPostProcessor;
+
+/**
+ * @author Jérôme Wacongne &lt;ch4mp@c4-soft.com&gt;
+ * @since 5.2.0
+ */
+public class AccessTokenRequestPostProcessor extends AccessTokenAuthenticationBuilder<AccessTokenRequestPostProcessor>
```

Comment:
PRs build on each other, adding content step by step. Initial order I chose differs from your expectation: access-token and open-id servlet implementation coming too early.
Will change order so that you have only jwt servlet and reactive in that branch and delete all other PRs

Reply:
You should now have servlet & reactive JWT flow APIs with nothing more (hopefully, if I didn't get mistaken somewhere)